crypto/stack/stack.c

```diff
@@ -248,7 +275,7 @@ static int internal_find(OPENSSL_STACK *st, const void *data,
     const void *r;
     int i;
 
-    if (st == NULL)
+    if (st == NULL || st->num <= 0)
```

Comment:
Change `<=` to `==`, that's at least consistent with all other places where you check `st->num`